10.0/contrib/wfcfg/standalone.xml

```diff
@@ -187,7 +187,7 @@
             </datasources>
         </subsystem>
         <subsystem xmlns="urn:jboss:domain:deployment-scanner:2.0">
-            <deployment-scanner path="deployments" relative-to="jboss.server.base.dir" scan-interval="5000" runtime-failure-causes-rollback="${jboss.deployment.scanner.rollback.on.failure:false}"/>
+            <deployment-scanner path="deployments" relative-to="jboss.server.base.dir" scan-interval="250" auto-deploy-zipped="${hot.deploy.enabled:false}" runtime-failure-causes-rollback="${jboss.deployment.scanner.rollback.on.failure:false}"/>
```

Comment:
Actually though - this is the time it takes between check for changes in the files. My goal with this is to minimize the delay in picking up the change. Basically our Docker image should be acting like the AppServer when it is launched by the IDE